hvplot/plotting/core.py

```diff
@@ -632,17 +668,30 @@ def image(self, x=None, y=None, z=None, colorbar=True, **kwds):
         """
         Image plot
 
+        You can very often use `image` to display an x, y grid, like for example geographic
+        data with `latitude` and `longitude` coordinates.
+
+        Reference: https://hvplot.holoviz.org/reference/xarray/image.html
+
+        Example:
+
+        >>> import hvplot.xarray
+        >>> import xarray as xr
+        >>> ds = xr.tutorial.open_dataset('air_temperature')
+        >>> ds.hvplot.image(x='lon', y='lat', z='air', groupby='time', cmap='kbc_r')
+
         Parameters
         ----------
-        x, y : string, optional
-            The coordinate variable along the x- and y-axis
+        x : string, optional
```

Comment:
Do we know these have to be strings. What if it is a `DataFrame` with `int` column names?

Reply:
Would you be happy if I added type annotations? 

`def image(self, x: str | None=None, y: str | None=None, z: str | None=None, colorbar: bool=True, **kwds):`

Reply:
That is a question that deserves its own issue :)

Reply:
Definitely a separate discussion, issue and PR. I won't object to typing hvPlot though.